Lib/test/test_tempfile.py

```diff
@@ -982,6 +982,22 @@ def test_basic(self):
         f = self.do_create(max_size=100, pre="a", suf=".txt")
         self.assertFalse(f._rolled)
 
+    def test_is_iobase(self):
+        # SpooledTemporaryFile should implement io.IOBase
+        self.assertIsInstance(self.do_create(), io.IOBase)
+
+    def test_iobase_interface(self):
```

Comment:
Those tests are not very interesting. It would be better to test the methods operate properly.

Reply:
It *literally* delegates everything to the underlying file.

The existing tests should cover the behaviors. The new behaviors are intended to flesh out the API so that `SpooledTemporaryFile`, and these tests seem to cover that.